google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -996,6 +996,155 @@ def iter_crm_folders(self, parent_id):
         for folder in resources:
             yield folder
 
+    def iter_kubernetes_nodes(self, project_id, zone, cluster):
+        """Iterate k8s nodes in an organization from Cloud Asset data.
+
+        Args:
+            project_id (str): id of the project to query.
+            zone (str): The zone the cluster is in.
+            cluster (str): The cluster name.
+
+        Yields:
+            dict: Generator of nodes.
+        """
+        resources = self.dao.iter_cai_assets(
+            ContentTypes.resource,
+            'k8s.io/Node',
+            '//container.googleapis.com/projects/{}/zones/{}/clusters/{}'
+            .format(project_id, zone, cluster),
+            self.session)
+        for node in resources:
+            yield node
+
+    def iter_kubernetes_pods(self, project_id, zone, cluster, namespace):
```

Comment:
For the iterators under Namespace, you need to include the k8s in the path. For example:

```
'//container.googleapis.com/projects/{}/zones/{}/clusters/{}/'
'k8s/namespaces/{}'.format(project_id, zone, cluster, namespace),
```

Reply:
Done